modules/build/src/main/scala/scala/build/preprocessing/directives/UsingScalaVersionDirectiveHandler.scala

```diff
@@ -17,24 +23,51 @@ case object UsingScalaVersionDirectiveHandler extends UsingDirectiveHandler {
   )
 
   def keys = Seq("scala")
+
   def handleValues(
     directive: StrictDirective,
     path: Either[String, os.Path],
     cwd: ScopePath,
     logger: Logger
   ): Either[BuildException, ProcessedUsingDirective] = {
-    val values        = directive.values
-    val scalaVersions = DirectiveUtil.stringValues(values, path, cwd)
-    if (scalaVersions.isEmpty)
-      Left(new NoScalaVersionProvidedError)
+    val numOfValues = directive.values.length
+    if (numOfValues > 1)
```

Comment:
We will use `// using scala 3.1 2.13 2.12` for cross compilation of libraries if I am not mistaken

cc @alexarchambault

Reply:
using directives syntax cannot detect multiple values that are not comma separated! So it detects:
`//> using scala 3.1, 2.13, 2.12`